.env.example

```
@@ -70,3 +70,12 @@ CARD_API_KEY=sda
   GMAIL_CLIENT_ID=your-client-id
   GMAIL_CLIENT_SECRET=your-client-secret
   GMAIL_FROM_EMAIL=your-email@gmail.com
+
+# Docker Configuration
+DOCKER_IMAGE_NAME=laravel-app
+DOCKER_REGISTRY=localhost:5000
+WWWUSER=1000
+WWWGROUP=1000
+APP_PORT=80
+FORWARD_DB_PORT=5432
+FORWARD_REDIS_PORT=6379
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)